python/cugraph-equivariant/cugraph_equivariant/_version.py

```diff
@@ -10,18 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+#
 
 import importlib.resources
 
-# Read VERSION file from the module that is symlinked to VERSION file
-# in the root of the repo at build time or copied to the module at
-# installation. VERSION is a separate file that allows CI build-time scripts
-# to update version info (including commit hashes) without modifying
-# source files.
 __version__ = (
-    importlib.resources.files("cugraph_equivariant")
-    .joinpath("VERSION")
-    .read_text()
-    .strip()
+    importlib.resources.files(__package__).joinpath("VERSION").read_text().strip()
 )
-__git_commit__ = ""
+try:
+    __git_commit__ = (
+        importlib.resources.files(__package__)
+        .joinpath("GIT_COMMIT")
+        .read_text()
+        .strip()
+    )
+except FileNotFoundError:
+    __git_commit__ = ""
+
+__all__ = ["__version__", "__git_commit__"]
```

Comment:
If you do push another commit here, could you please re-order this (and all the similar `__all__` constants) so it's in alphabetical order? Just so it'd be a little easier to read at a glance if it grows in the future.